campaigns/fhv/10_the_second_evening.json

```diff
@@ -1436,7 +1436,16 @@
         ]
       }
     },
-
+    {
+      "id": "helen_peters_joined_the_survey_story",
+      "narration": {
+        "id": "the_feast_of_hemlock_vale.the_second_evening.codex_omega_helen_peters_2",
```

Comment:
I assume this exists, as part 1 and 3 are referenced already, but have no clue whether that's actually the case

Reply:
It exists in Russian and English, which are the two that declare it.  So hitting this url 
https://static.arkhamcards.com/audio/LANG_CODE/the_feast_of_hemlock_vale.the_second_evening.codex_omega_helen_peters_2.mp3 is the way to test.

This won't work for english which has its own auth system (that I don't control)